FeedSX/screens/UniversalFeed/index.tsx

```diff
@@ -109,23 +129,83 @@ const UniversalFeed = () => {
     getInitialData();
   }, [navigationRef, lmFeedClient]);
 
-  // todo: dummy data
-  // readonly props consumed by UI component
-  const props: PostStateProps = {
-    showBookMarkIcon: true,
-    showShareIcon: true,
-    footerTextStyle: {},
-    showLabel: true,
-    labelType: 'Admin',
-  };
-
   // this calls the getFeed api whenever the page number gets changed
   useEffect(() => {
     if (accessToken) {
       fetchFeed();
     }
   }, [accessToken, feedPageNumber]);
 
+  // this function closes the post action list modal
+  const closePostActionListModal = () => {
+    setShowActionListModal(false);
+  };
+
+  // this function handles the functionality on the pin option
+  const handlePinPost = async (id: string) => {
+    let payload = {
+      postId: id,
+    };
+    dispatch(pinPostStateHandler(payload.postId) as any);
+    let pinPostResponse = await dispatch(
+      pinPost(
+        PinPostRequest.builder().setpostId(payload.postId).build(),
```

Comment:
move `setpostId()` and `build()` to next line to maintain readability of the code.

Reply:
resolved